CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
-## v1.2.20 - ?
+## v1.3.0 - ?
```

Comment:
Shouldn't we bump to `2.0.0` ? as this is strictly breaking?

Reply:
I discussed it with @bartv earlier today and we were of the opinion that we can get away with a minor, provided that the fallout remains minimal. Because if we make this a major, we also have to make the change on std a major, and then we have to diverge iso6 and master for the std module. If you think differently, we can discuss it.

Reply:
I would like to start with a dev version as a minor and see what the fallout is (if any at all)

Reply:
The type is still a string just a slightly different constraint